src/chromeDebugAdapter.ts

```diff
@@ -111,6 +116,12 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         return super.attach(args);
     }
 
+    public configurationDone(): Promise<void> {
+        // This means all the setBreakpoints requests have been completed. So we can navigate to the original file/url.
+        this.chrome.Page.navigate({url: this._userRequestedUrl});
```

Comment:
This means all the requests have been sent, but not necessarily handled by the adapter. I'm not sure if that's an issue or not. Should this be waiting for us to finish handling the initial setBreakpoints?

Reply:
What do you mean by not handled by the adapter? As long as Chrome receives all the requests before we navigate we should be good. Wouldn't that be the case here?

Reply:
I guess so? I'm just wondering whether it would be a problem if we do something like

- Get setBreakpoints
- Send lots of setBreakpoint requests to Chrome
- Don't get responses yet from Chrome
- Tell Chrome to navigate

Maybe Chrome will finish processing the set breakpoint requests before it actually navigates. Or maybe there's a race case where it could load the new page before it's actually set the breakpoints, I'm not sure.

Reply:
Hmm. Is there a way where we can be sure that Chrome has finished processing those requests?

Reply:
All I can think of is waiting on the responses from Chrome. But I think this is similar to what we do in node2 already, and I don't wait before doing a 'continue' and it's never caused a problem. So I think this is fine for now.

Reply:
Sounds good!